README.md

```diff
@@ -33,35 +33,6 @@ git clone --depth=1 https://github.com/alvesjtiago/hover-preview.git
 
 Or manually create a folder named "hover-preview" on your Packages folder and copy the content of this repo to it.
 
-## Configuration
-
-#### max_dimensions:
-
-- max\_width x max\_height for which the pop-up is considered too big and should be resized, (default: `[320, 240]`).
-
-#### search_mode: 
```

Comment:
Why were these settings removed from the README?

Reply:
I removed all the configurations section from the README in 301e66f4631b1846ab6b2c6e2154f919a241eb94 because they are already well explained in the settings file, and because other plugins rarely include a configurations sections in their READMEs.
This specific section was later removed from the settings file too in 
cf84ee8 along with all the resize logic.